src/devices/Card/Ndef/samples/Program.cs

```diff
@@ -411,7 +412,7 @@ Pn5180 Ft4222Pn5180()
         Console.WriteLine();
     }
 
-    var (chip, dll) = FtCommon.GetVersions();
+    var (chip, dll) = Ft4222Common.GetVersions();
```

Comment:
could FtCommon enumerate both Ft4222s and Ft232?

Reply:
because FT4222 have a specific set of API, the GetVersions is only available with FT4222.